CoEpi/api/CoEpiApi.swift

```diff
@@ -17,10 +17,10 @@ class CoEpiApiImpl: CoEpiApi {
 //    private let baseUrl = "https://q69c4m2myb.execute-api.us-west-2.amazonaws.com/v3/"
     private let baseUrl = "https://v1.api.coepi.org/"
 
-    func postCenReport(cenReport: MyCenReport) -> Completable {
-        os_log("Sending CEN report to API: %@", log: servicesLog, type: .debug, "\(cenReport)")
+    func postCenReport(myCenReport: MyCenReport) -> Completable {
```

Comment:
Personal preference: I prefer to keep parameter names simple when it's obvious from context. Ideally this would be called only `report`. But you can let it like this.